test/src/test/java/jenkins/model/JenkinsBuildsAndWorkspacesDirectoriesTest.java

```diff
@@ -258,6 +286,13 @@ private boolean logWasFound(String searched) {
                 .anyMatch(record -> record.getMessage().contains(searched));
     }
 
+	private boolean logWasFoundAtLevel(String searched, Level level) {
+		return loggerRule.getRecords().stream()
+				.filter(record -> record.getMessage().contains(searched))
+				.filter(record -> record.getLevel().equals(level))
+				.collect(Collectors.toList()).size() > 0;
```

Comment:
NIT: see `anyMatch()` above. It's somehow unimportant here in test context, but it **is** important in general. Because what the code here is doing necessitates to go computing the whole list size. 

I.e. if `Collectors.toList()` is actually going to compute a list of billions of elements, it could take a lot of time, when anyMatch will just stop on the first match and yield the return result.